src/engine/renderer/tr_shader.cpp

```diff
@@ -5992,209 +6000,192 @@ A second parameter will cause it to print in sorted order
-		{
-			str += "SS_PORTAL           ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_DEPTH) )
-		{
-			str += "SS_DEPTH            ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_ENVIRONMENT_FOG) )
-		{
-			str += "SS_ENVIRONMENT_FOG  ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_ENVIRONMENT_NOFOG) )
-		{
-			str += "SS_ENVIRONMENT_NOFOG";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_OPAQUE) )
-		{
-			str += "SS_OPAQUE           ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_DECAL) )
-		{
-			str += "SS_DECAL            ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_SEE_THROUGH) )
-		{
-			str += "SS_SEE_THROUGH      ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_BANNER) )
-		{
-			str += "SS_BANNER           ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_FOG) )
-		{
-			str += "SS_FOG              ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_UNDERWATER) )
-		{
-			str += "SS_UNDERWATER       ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_WATER) )
-		{
-			str += "SS_WATER            ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_FAR) )
-		{
-			str += "SS_FAR              ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_MEDIUM) )
-		{
-			str += "SS_MEDIUM           ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_CLOSE) )
-		{
-			str += "SS_CLOSE            ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_BLEND0) )
-		{
-			str += "SS_BLEND0           ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_BLEND1) )
-		{
-			str += "SS_BLEND1           ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_BLEND2) )
-		{
-			str += "SS_BLEND2           ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_BLEND3) )
-		{
-			str += "SS_BLEND3           ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_BLEND6) )
-		{
-			str += "SS_BLEND6           ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_ALMOST_NEAREST) )
-		{
-			str += "SS_ALMOST_NEAREST   ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_NEAREST) )
-		{
-			str += "SS_NEAREST          ";
-		}
-		else if ( shader->sort == Util::ordinal(shaderSort_t::SS_POST_PROCESS) )
-		{
-			str += "SS_POST_PROCESS     ";
-		}
-		else
-		{
-			str += "                    ";
-		}
+			stagePropertyString += padStringSuffix( foundStageTypeString, maxStageTypeStringLen );
+			stagePropertyString += separator;
 
-		if ( shader->interactLight )
-		{
-			str += "IA ";
-		}
-		else
-		{
-			str += "   ";
-		}
-
-		if ( shader->defaultShader )
-		{
-			Log::Notice("%s: %s (DEFAULTED)", str.c_str(), shader->name );
+			Log::Notice("%s%i:%s", stagePropertyString, j, shaderNameString );
```

Comment:
Use `Log::CommandInteractionMessage` to avoid log suppression.

Reply:
Done. In the process I noted `Log::CommandInteractionMessage` doesn't support string formatting, so I used `Str::Format` to build strings printed with `Log::CommandInteractionMessage`.